extensions/resource-deployment/package.json

```diff
@@ -1,368 +1,370 @@
+								"displayName": "%bdc-deployment-target-new-aks%"
+							},
+							{
+								"name": "existing-aks",
+								"displayName": "%bdc-deployment-target-existing-aks%"
+							},
+							{
+								"name": "existing-kubeadm",
+								"displayName": "%bdc-deployment-target-existing-kubeadm%"
+							}
+						]
+					}
+				],
+				"providers": [
+					{
+						"wizard": {
+							"type": "new-aks",
+							"notebook": "%bdc-2019-aks-notebook%"
+						},
+						"requiredTools": [
+							{
+								"name": "kubectl"
+							},
+							{
+								"name": "azure-cli"
+							},
+							{
+								"name": "azdata",
+								"version": "15.0.2070"
+							}
+						],
+						"when": "target=new-aks&&version=bdc2019"
+					},
+					{
+						"wizard": {
+							"type": "existing-aks",
+							"notebook": "%bdc-2019-existing-aks-notebook%"
+						},
+						"requiredTools": [
+							{
+								"name": "kubectl"
+							},
+							{
+								"name": "azdata"
+							}
+						],
+						"when": "target=existing-aks&&version=bdc2019"
+					},
+					{
+						"wizard": {
+							"type": "existing-kubeadm",
+							"notebook": "%bdc-2019-existing-kubeadm-notebook%"
+						},
+						"requiredTools": [
+							{
+								"name": "kubectl"
+							},
+							{
+								"name": "azdata"
+							}
+						],
+						"when": "target=existing-kubeadm&&version=bdc2019"
+					}
+				],
+				"agreement": {
+					"template": "%bdc-agreement%",
+					"links": [
+						{
+							"text": "%bdc-agreement-privacy-statement%",
+							"url": "https://go.microsoft.com/fwlink/?LinkId=853010"
+						},
+						{
+							"text": "%bdc-agreement-bdc-eula%",
+							"url": "https://go.microsoft.com/fwlink/?LinkId=2002534"
+						},
+						{
+							"text": "%bdc-agreement-azdata-eula%",
+							"url": "https://aka.ms/azdata-eula"
+						}
+					]
+				}
+			},
+			{
+				"name": "sql-windows-setup",
+				"displayIndex": 1,
+				"displayName": "%resource-type-sql-windows-setup-display-name%",
+				"description": "%resource-type-sql-windows-setup-description%",
+				"platforms": [
+					"win32"
+				],
+				"icon": {
+					"light": "./images/sql_server_on_windows.svg",
+					"dark": "./images/sql_server_on_windows_inverse.svg"
+				},
+				"options": [
+					{
+						"name": "version",
+						"displayName": "%version-display-name%",
+						"values": [
+							{
+								"name": "sql2017",
+								"displayName": "%sql-2017-display-name%"
+							},
+							{
+								"name": "sql2019",
+								"displayName": "%sql-2019-display-name%"
+							}
+						]
+					}
+				],
+				"providers": [
+					{
+						"downloadUrl": "https://go.microsoft.com/fwlink/?linkid=853016",
+						"requiredTools": [],
+						"when": "version=sql2017"
+					},
+					{
+						"webPageUrl": "https://www.microsoft.com/evalcenter/evaluate-sql-server-2019-rc",
+						"requiredTools": [],
+						"when": "version=sql2019"
+					}
+				]
+			}
+		]
+	},
+	"dependencies": {
+		"getos": "^3.1.1",
```

Comment:
I see in the code we are currently using `os.platform()` to detect the platform.  Is there a reason we are adding a new node dependency for apparently the same purpose?  Ideally, we'll keep our external dependencies to a minimum.

Reply:
platform does not give the linux distribution details.